apps/vs-code-extension/project.json

```diff
@@ -16,13 +16,23 @@
         "external": ["vscode"],
         "platform": "node",
         "format": ["cjs"],
-        "esbuildOptions": {
-          "sourcemap": "linked"
-        },
         "assets": [
           { "input": "apps/vs-code-extension/assets", "glob": "*", "output": "/" }
         ]
-      }
+      },
+      "configurations": {
+        "dev": {
+          "esbuildOptions": {
+            "sourcemap": "linked"
+          },
+          "minify": false
+        },
+        "prod": {
+          "sourcemap": false,
```

Comment:
different from `dev` (esbuildOptions)

Reply:
Yes, but it's desired to have sourcemaps only during development and not in production. See the Langium guide on code bundling for more details: https://langium.org/guides/code-bundling/

Reply:
Sorry, I think I got you wrong. You mean `exbuildOptions { "sourcemap": "linked" }` vs just `"sourcemap": false`?

Reply:
Should be addressed via [78875e6](https://github.com/jvalue/jayvee/pull/342/commits/78875e62cc6c6bd6aea34cc397e7196faf6939f7)